### PR TITLE
[dapp-high-roller] Show User and Opponent from State in AppGame

### PR DESCRIPTION
**Pull request by Alonski**
### Description

This adds opponent info to app state.
It is passed in as opponent and user data into app-game for the UI.

- [x] Deploy preview is functional

### CODE REVIEW

File: packages/playground-server/.env-cmdrc.local [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "development": {
+    "DB_ENGINE": "pg",
+    "DB_CONNECTION_STRING": "postgresql://playground_user:GFnggtxysjRjKJ3AN8g6cTYg@104.248.239.2:5432/playground_db",
```
Comment (ebryn) [inline packages/playground-server/.env-cmdrc.local]: ... staging env password shouldn't just be in plain text here

File: packages/dapp-high-roller/src/components/app-root/app-root.tsx [hunk 2]
```diff
@@ -8,6 +8,9 @@ import { cf, Node } from "../../data/types";
 declare var NodeProvider;
 declare var cf;
 
+const AUTH_TOKEN =
+  "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY4NzI5N2JjLTgwMTQtNGM4Mi04Y2VlLTNiN2NhN2RiMDlkNCIsInVzZXJuYW1lIjoiQWxvbnNraTIiLCJhZGRyZXNzIjoiMHhjNjBiOTAyM2JiOGRjMTUzYjQwNDY5NzczMjhjZTc5YWYxMmE3N2UwIiwiZW1haWwiOiJhbG9uMkBleGFtcGxlLmNvbSIsIm11bHRpc2lnQWRkcmVzcyI6IjB4OTQ5OWFjNUE2NmMzNjQ0N2U1MzVkMjUyYzA0OTMwNEQ4MDk2MUNFRCIsImlhdCI6MTU0NzQwNDUzNSwiZXhwIjoxNTc4OTYyMTM1fQ.innx0udzB2fj2_ZWYGugDJlz-Wb-qSckjqjyIBBTyS4";
```
Comment (ebryn) [inline packages/dapp-high-roller/src/components/app-root/app-root.tsx]: this shouldn't be necessary anymore as we posted a postMessage API for retrieving a token

File: packages/playground-server/.env-cmdrc.local (hunk 1 above)
Comment (Alonski) [reply packages/playground-server/.env-cmdrc.local]: Deleted it

File: packages/dapp-high-roller/src/components/app-root/app-root.tsx (hunk 2 above)
Comment (Alonski) [reply packages/dapp-high-roller/src/components/app-root/app-root.tsx]: Deleted it

File: packages/dapp-high-roller/src/components/app-game/app-game.tsx [hunk 3]
```diff
@@ -130,15 +131,16 @@ export class AppGame {
...
-  @Prop({ mutable: true }) myName: string = "Facundo";
   @Prop({ mutable: true }) betAmount: string = "3 ETH";
-  @Prop({ mutable: true }) opponentName: string = "John";
   @Prop({ mutable: true }) isProposing: boolean = false;
   @Prop({ mutable: true }) appInstanceId: string = "";
 
   @Prop() cfProvider: cf.Provider = {} as cf.Provider;
   @Prop({ mutable: true }) appInstance: AppInstance = {} as AppInstance;
 
+  @Prop() user: any = { username: "Facundo" };
```
Comment (joelalejandro) [inline packages/dapp-high-roller/src/components/app-game/app-game.tsx]: @Alonski Should probably have empty strings?

File: packages/dapp-high-roller/src/components/app-provider/app-provider.tsx [hunk 4]
```diff
@@ -35,7 +32,7 @@ export class AppProvider {
     this.nodeProvider = params.get("standalone")
       ? new MockNodeProvider()
       : new NodeProvider();
-    return this.nodeProvider.connect().then(this.setupCfProvider.bind(this));
+    return this.nodeProvider.connect().then(this.setupCfProvider.bind(this)); // TODO have this block rendering
```
Comment (joelalejandro) [inline packages/dapp-high-roller/src/components/app-provider/app-provider.tsx]: @Alonski I suggest you have a look to how this was implemented on the `dapp-tic-tac-toe` package. Basically you don't render anything until `nodeProvider.isConnected === true`.

File: packages/dapp-high-roller/src/components/app-game/app-game.tsx (hunk 3 above)
Comment (Alonski) [reply packages/dapp-high-roller/src/components/app-game/app-game.tsx]: But if the state isn't injected? Maybe these should be generic like "Bob" and "Alice"?

**Comment by Alonski** [comment]
I am not sure exactly how to fix failing tests hmm
`ReferenceError: window is not defined`
In `app-root`
It never yelled about this before

**Comment by Alonski** [comment]
@ebryn I feel this is ready for review and merging

**Pull request merged**